attestation-agent/README.md

```diff
@@ -0,0 +1,140 @@
+# Attestation Agent
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fattestation-agent.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fattestation-agent?ref=badge_shield)
```

Comment:
We may need to consolidate to one FOSSA thing as well? I think we should probably have a top level README and maybe that could include the scanner (with the link to the image-rs one).